src/main/resources/anniv5Resources/localization/eng/Potionstrings.json

```
@@ -0,0 +1,34 @@
+{
+  "${ModID}:BoosterBrew": {
+    "NAME": "Booster Brew",
+    "DESCRIPTIONS": [
+      "Choose 1 of 3 random cards from different Packs. Add #b1 copy of it to your hand. It costs 0 this turn.",
```

Need highlighting for the numbers.
```suggestion
      "Choose #b1 of #b3 random cards from different Packs. Add #b1 copy of it to your hand. It costs 0 this turn.",
```